### PR TITLE
move definitions of createSmallDeformationProcess() to cpp file

### PR DESCRIPTION
**Pull request by norihiro-w**
This PR is for speeding up build times and following the way of other processes

### CODE REVIEW

**Comment by norihiro-w** [comment]
@bilke Jenkins test nr. 4 is ok, but GitHub points to nr. 3. Should I run Jenkins again?

File: ProcessLib/SmallDeformation/CreateSmallDeformationProcess.h [hunk 1]
```diff
@@ -10,95 +10,21 @@
...
 
 namespace ProcessLib
 {
 namespace SmallDeformation
 {
-template <int DisplacementDim>
-class SmallDeformationProcess;
-
-extern template class SmallDeformationProcess<2>;
-extern template class SmallDeformationProcess<3>;
```
Comment (chleh) [inline ProcessLib/SmallDeformation/CreateSmallDeformationProcess.h]: Maybe the extern template specifications should stay in the header?

Comment (norihiro-w) [inline ProcessLib/SmallDeformation/CreateSmallDeformationProcess.h]: so far, it's unnecessary. ProjectData needs only the base class Process.

Comment (chleh) [inline ProcessLib/SmallDeformation/CreateSmallDeformationProcess.h]: OK.

**Comment by norihiro-w** [comment]
Jenkins, test this please

**Pull request merged**